std_msgs/Float32 thrust
std_msgs/Float32 torque_x
std_msgs/Float32 torque_y
std_msgs/Float32 torque_z
